src/VisualStudio/Core/Def/Experimentation/VisualStudioExperimentationService.cs

```diff
@@ -25,11 +26,13 @@ public VisualStudioExperimentationService(IThreadingContext threadingContext, SV
         {
             object experimentationServiceOpt = null;
             MethodInfo isCachedFlightEnabledInfo = null;
+            IVsFeatureFlags featureFlags = null;
 
             threadingContext.JoinableTaskFactory.Run(async () =>
             {
                 try
                 {
```

Comment:
curious. why do we need this reflection and JTF?

Reply:
no answer for this except those try/catch? this is way more smelly than try/catch which impact is quite apparent.